utils/env_preferences/tests/parse/windows.rs

```diff
@@ -22,16 +22,16 @@ fn collation() {
     const CASES: [(&str, &str); 12] = [
         ("de-DE_phoneb", "de-DE-u-co-phonebk"),
         ("es-ES_tradnl", "es-ES-u-co-trad"),
-        ("ja-JP_radstr", "ja-JP-u-co-stroke"),
+        ("ja-JP_radstr", "ja-JP-u-co-unihan"),
         ("zh-CN_phoneb", "zh-CN-u-co-phonebk"),
         ("zh-CN_stroke", "zh-CN-u-co-stroke"),
-        ("zh-HK_radstr", "zh-HK-u-co-stroke"),
-        ("zh-MO_radstr", "zh-MO-u-co-stroke"),
+        ("zh-HK_radstr", "zh-HK-u-co-unihan"),
+        ("zh-MO_radstr", "zh-MO-u-co-unihan"),
         ("zh-MO_stroke", "zh-MO-u-co-stroke"),
         ("zh-SG_phoneb", "zh-SG-u-co-phonebk"),
```

Comment:
Side note: We should find out what Singapore phonebook order really is. (Not for this changeset.)

Reply:
Or is `zh-SG_phoneb` just a test case and not something that Windows actually does?

Reply:
It might be worth mentioning thata all of those test cases are extracted directly from the [MS-LCID](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f) document (specifically final table in section 2.2). There's a comment in `tests/dataset/mod.rs` that provides attribution, but I must have forgot to do it here.